pkg/runtime/wfengine/wfengine_test.go

```diff
@@ -760,7 +760,9 @@ func startEngine(ctx context.Context, r *task.TaskRegistry) (backend.TaskHubClie
 }
 
 func getEngine() *wfengine.WorkflowEngine {
-	engine := wfengine.NewWorkflowEngine()
+	engine := wfengine.NewWorkflowEngine(&wfengine.WFConfig{
+		AppID: "test",
+	})
```

Comment:
in the line below https://github.com/dapr/dapr/pull/6192/files#diff-acb2ead1c43373f11db66d9821d6bfa8b1e16ee5566bc89fb0936cedefcd4019R781
Please do not panic, fail the test after planning in `t` to this method.